0001-typedb/01-start/03-quickstart.md

```diff
@@ -0,0 +1,673 @@
+---
+pageTitle: Quickstart guide
+keywords: getting started, typedb, typeql, tutorial, quickstart, overview
+longTailKeywords: get started with typedb, typedb tutorial, typedb quickstart, learn typedb
+summary: Learn how to create a TypeDB database, load schema and data, perform queries.
+toc: false
+---
+
+# Quickstart guide
+
+## Overview
+
+This Quickstart guide goes through the step-by-step process of creating and interacting with a TypeDB database designed 
+for an [Identity and Access Management](https://en.wikipedia.org/wiki/Identity_management) solution that will be 
+described [later](04-iam-schema.md).
+
+<div class="note">
+[Important]
+We highly recommend completing this guide. Its goal is to prepare an environment for TypeDB exploration and development.
+The resulting database (schema and data) will be needed for example queries throughout the documentation.
+</div>
+
+## Prerequisites
+
+This Quickstart guide takes advantage of TypeDB Studio, a desktop GUI tool for interacting with TypeDB. To proceed, 
+please install both TypeDB and TypeDB Studio:
+
+- [Install TypeDB](02-installation.md)
+- [Install TypeDB Studio](../../02-clients/01-studio.md#get-typedb-studio)
+
+IMPORTANT:
+<div class="note">
+[Important]
+It’s recommended to use TypeDB and TypeDB Studio of the same version number in order to ensure compatibility. If 
+TypeDB Studio doesn't have a version similar to the TypeDB release — use the closest lower version instead.
```

Comment:
These sentences aren't very hopeful - we should do what we can to avoid a user ever encountering this scenario, but we might need to provide further detail: if TypeDB Studio is absolutely necessary, they may need to downgrade their version of TypeDB for Studio compatibility.

Reply:
Downgrading an existing DB installation just because the IDE to work with it didn't updated yet - is less than ideal. Looks very bad on us. 

Also the text complies with recommendations I got in Discord AND with our old docs on that matter - https://docs.vaticle.com/docs/studio/overview#version-compatibility.

Reply:
Alright, let's provide support for this on a case by case basis.

Reply:
Sorry, I'm not sure I follow. What else can we provide if we already have direct instructions on what to do AND a separate compatibility table? I can add the link to the table here if you want.